cmake/sysbuild/suit.cmake

```diff
@@ -103,10 +103,12 @@ endfunction()
 #   'output_file' - path to output signed envelope
 function(suit_sign_envelope input_file output_file)
   cmake_path(GET ZEPHYR_NRF_MODULE_DIR PARENT_PATH NRF_DIR_PARENT)
-  sysbuild_get(CONFIG_SUIT_ENVELOPE_SIGN_SCRIPT IMAGE ${DEFAULT_IMAGE} VAR CONFIG_SUIT_ENVELOPE_SIGN_SCRIPT KCONFIG)
-  suit_set_absolute_or_relative_path(${CONFIG_SUIT_ENVELOPE_SIGN_SCRIPT} ${NRF_DIR_PARENT} SIGN_SCRIPT)
+  if(NOT DEFINED SB_CONFIG_SUIT_ENVELOPE_SIGN_SCRIPT)
```

Comment:
You may also consider not calling `suit_sign_envelope` if the script is not defined. For me - both solutions are fine.